src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -12,7 +12,7 @@
 
 'use strict';
 
-import type {Fiber} from 'ReactFiber';
+import type { Fiber } from 'ReactFiber';
```

Comment:
Hey @acdlite – this project uses prettier to format all JavaScript code. Mind running `yarn prettier` and amending the changes to this PR?